dist/Data-Dumper/Dumper.xs

```diff
@@ -1733,6 +1733,8 @@ Data_Dumper_Dumpxs(href, ...)
 		croak("Call to new() method failed to return HASH ref");
 	    if (gimme != G_ARRAY)
 		XPUSHs(sv_2mortal(retval));
+            else if (imax < 0) /* Would leak if there was nothing to dump */
```

Comment:
Looks like this is formatted incorrectly.

Reply:
No, it's a tabs vs spaces, the `if (gimme...` is indented with a tab, while the else is just spaces.